nginx-controller/controller/controller.go

```diff
@@ -979,6 +995,45 @@ func (lbc *LoadBalancerController) getTargetPort(svcPort *api_v1.ServicePort, sv
 	return portNum, nil
 }
 
+func (lbc *LoadBalancerController) getEndpointsInfo(
+	svc *api_v1.Service,
+) ([]nginx.EndpointInfo, error) {
```

Comment:
This function declaration is not long enough to wrap it into multiple lines.

Reply:
Line is long enough to not fit into 80 symbols per line. It's studied, that optimal line length should be <80 for balance of comprehension and reading speed — https://en.wikipedia.org/wiki/Line_length

Telling you as contributor to that code, it's very difficult to read with that long lines which even do not fit my editor, so I need to use horizontal scrolling.

However, I can join lines if you disagree.

Reply:
Consistency is the key. The other function declarations of similar length are one-line.